python/example-client-workflow.ipynb

```diff
@@ -0,0 +1,885 @@
+      "\n",
+      "Dataset Query Details: Successfully fetched data for handle '10378.1/1908974'\n",
+      "\n",
+      "Dataset Display Name: TEst\n"
+     ]
+    }
+   ],
+   "source": [
+    "\n",
+    "dataset = await client.datastore.fetch_dataset(id = \"10378.1/1908974\")\n",
+    "\n",
+    "print(dataset) # Fetched dataset pythonic object.\n",
+    "print()\n",
+    "print(\"Dataset Query Details:\", dataset.status.details) # Accessing fetched dataset query details\n",
+    "print()\n",
+    "print(\"Dataset Display Name:\", dataset.item.display_name) # Accessing fetched dataset name"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ProvenaInterfaces.RegistryModels import *\n",
+    "\n",
+    "dataset_to_create = CollectionFormat(\n",
+    "        associations=CollectionFormatAssociations(\n",
+    "        organisation_id=\"10378.1/1893860\",\n",
+    "        data_custodian_id=\"10378.1/1893843\",\n",
+    "        point_of_contact= None\n",
+    "        ),\n",
+    "        approvals=CollectionFormatApprovals(\n",
+    "            ethics_registration = DatasetEthicsRegistrationCheck(relevant=False, obtained=False),\n",
+    "            ethics_access=DatasetEthicsAccessCheck(relevant= False, obtained= False),\n",
+    "            indigenous_knowledge=IndigenousKnowledgeCheck(relevant=False, obtained= False),\n",
+    "            export_controls=ExportControls(relevant=False, obtained=False)\n",
+    "        ),\n",
+    "        dataset_info=CollectionFormatDatasetInfo(\n",
+    "            name=\"Parth testing\",\n",
+    "            description=\"testing dataset\",\n",
+    "            access_info=AccessInfo(reposited=True, uri=None, description=None),\n",
+    "            publisher_id=\"10378.1/1893860\",\n",
+    "            published_date=date.today(),\n",
+    "            license = \"https://www.google.com\", #type:ignore\n",
+    "            created_date=date.today(),\n",
```

Comment:
Since this PR was there was that migration of the publish and creation date fields causing this cell to fail not because of anyones fault. Can you please update these though so they can work? Thanks